scripts/setup

```diff
@@ -68,6 +68,9 @@ success_msg() {
 }
 
 
+export FLASK_SKIP_DOTENV=1  # avoid 'There are .env or .flaskenv…' messages
+export PYTHONWARNINGS="ignore::DeprecationWarning"
```

Comment:
Better to make this optional.

Reply:
I do it via a `--warnings` (or `-w`) option to enable warnings.